### PR TITLE
log timings of block body validation

### PR DESCRIPTION
**Pull request by arvidn**
I noticed that we don't use `time` in this file at all. Is that a deliberate choice because it's consensus critical?

### CODE REVIEW

File: chia/consensus/blockchain.py [hunk 1]
```diff
@@ -242,6 +244,8 @@ async def receive_block(
             fork_point_with_peak,
             self.get_block_generator,
         )
+        done = time.time()
+        log.debug(f"validate_block_body took {done-start} seconds")
```
Comment (mariano54) [inline chia/consensus/blockchain.py:248]: You might want to differentiate between the two calls, so we can see the differences.

Comment (arvidn) [reply chia/consensus/blockchain.py]: done

File: chia/consensus/blockchain.py [hunk 2]
```diff
@@ -534,6 +539,8 @@ async def validate_unfinished_block(
             None,
             self.get_block_generator,
         )
+        done = time.time()
```
Comment (Nikolaj-K) [inline chia/consensus/blockchain.py:542]: Probably a bit unconventional to denote the variable by an adjective/adverb.

**Comment by arvidn** [comment]
I think one reason we don't log anything in this function today is because it may be run in `ProcessPoolExecutor`, and the loggers aren't set up in those processes.

Also, there are already logs for "Block pre-validation time" and "Validation time for peak". So this patch may be redundant.

**Comment by mariano54** [comment]
I have included the logging improvements here also:
https://github.com/Chia-Network/chia-blockchain/pull/4844
So I think we can close this one

**Pull request closed without merging**